File: 921_Minimum_Add_to_Make_Parentheses_Valid.cpp
int count = 0;
    stack<string> stackObj;
    for (char c : s)
    {
        if (c == '(')
        {
            stackObj.push("(");
        }
        else
        {

            if (stackObj.empty())
            {
                count++;
            }
            else
            {
                stackObj.pop();
            }
        }

    }
    return count + stackObj.size();

File: 921_Minimum_Add_to_Make_Parentheses_Valid.py
class Solution:
    def minAddToMakeValid(self, s: str) -> int:
        myList = []
        count = 0
        for i in s :
            if i == "(":
                myList.append(i)
            else:
                if len(myList) == 0:
                    # return len(myList) - string.index(i) - 1
                    count += 1
                    
                else:
                    myList.pop()
        count += len(myList)
        return count
